README.md

```diff
@@ -8,9 +8,7 @@ at https://pros.cs.purdue.edu/.
 
 ## How are these docs built?
 
-The PROS docs are built with [Sphinx](http://www.sphinx-doc.org/en/master/). The
-theming is a modified version of the [Read The Docs theme](http://sphinx-rtd-theme.readthedocs.io/en/latest/),

```

Comment:
while this may be discussed later as well, I think it's still relevant information in this section and should not be removed.

Reply:
Ok. I agree with you. I'll move those back.